test/Main.hs

```diff
@@ -64,5 +67,18 @@ translationPasses =
     [ FrontendDesugar.allDesugar
     ]
 
+contractTests :: FilePath -> IO ()
+contractTests file = do
+  parsed <- readFile $ file
+  let parsedIdString = Parser.parse (encodeUtf32BE parsed)
+  case parsedIdString of
+    Left s -> putStrLn s
+    Right _t -> putStrLn $ "parsed " <> file
```

Comment:
The output console doesn't print out either of these outputs atm.

Reply:
Actually nvm. It does when I comment out the other tests and run `make test`. Both files parsed.